src/ore/src/tracing.rs

```diff
@@ -207,13 +207,15 @@ where
     let service_name = service_name.to_string();
 
     let config = config.into();
+    // See: https://no-color.org/
+    let no_color = std::env::var_os("NO_COLOR").unwrap_or_else(|| "".into()) != "";
```

Comment:
```suggestion
    let no_color = std::env::var_os("NO_COLOR") != None
```

Reply:
It also must be non-empty according to the website!